spec/controllers/pieces_controller_spec.rb

```diff
@@ -126,10 +126,29 @@
         piece.reload
         expect(piece.x_position).to eq(4)
       end
+
+      it 'advances the game to the next turn' do
+        piece = FactoryGirl.create(:queen, x_position: 2, y_position: 2, game_id: game2.id)
+          put :update, params:
```

Comment:
Lines 132 - 144 basically imitate what will happen when we drop a piece on the UI... they are the parameters for the HTTP request.